package.json

```diff
@@ -71,11 +71,8 @@
         "shallowequal": "1.1.0"
     },
     "peerDependencies": {
-        "@types/react": "17.0.39",
-        "@types/react-dom": "17.0.11",
-        "react": "17.0.2",
-        "react-app-polyfill": "3.0.0",
```

Comment:
presuming this isn't actually necessary as the polyfills should get added as needed in compilation